### ISSUE TITLE
[1.12-10] Astral Sorcery structure that gives random resources dupe bug

### ISSUE THREAD

**Issue by K-chris** [HellFirePvP/AstralSorcery#488]
when using the Actually additions Auto-breaker to break the middle block drops the item and collects it into the Auto-breaker inventory 
![32702504-9ca3e536-c7e8-11e7-9e2a-5dfe2b9a16e8](https://user-images.githubusercontent.com/6795797/34645903-97c78ba2-f30e-11e7-9046-4e761d58cb0b.png)

**Comment by Doomgull** [comment]
Need to clarify what you mean, because that was written ridiculously unclear: what is being broken? What is being dropped and collected? What is the dupe?

**Comment by K-chris** [comment]
In the underground structure that looks like that included photo. If you use the block breaker from actually additions to break the respawning ore block it will collect the ore into the actually additions block breaker inventory and it still drops the same ore onto the ground. So you get 2x the amount from the structure  Not sure if that is sufficiently explained now, maybe dupe bug is the wrong term.

**Comment by Doomgull** [comment]
I would recommend opening a ticket with AA to see what they're doing with it.

**Comment by HellFirePvP** [comment]
This issue was discussed internally and it was concluded to add additional checks to ensure it's a player mining this or is at least nearby. And if someone wants to go through the effort of building an autominer next to it for 200 ores and stand there until it has mined all of them, then please, have at it.

tl;dr: gonna need a player nearby but the problem described here is known and not fixable easily.
